### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by deining**
This PR fixes a few typos I spotted in the project.

## Summary by CodeRabbit

- **Documentation**
	- Corrected minor typographical errors across several documents to improve clarity and professionalism, including:
		- "specifiy" to "specifying"
		- "comma seperated" to "comma separated"
		- "application" spelling corrections
		- "doesnt" to "doesn't"
		- "Miminal" to "Minimal"
		- "classs" to "class"
		- "envrionment" to "environment"
		- "listeneres" to "listeners"
		- "tha" to "the" in print dialog description.
- **New Features**
	- Windows-specific enhancements include restricting `ZoomFactor` and `IsZoomControlEnabled` options to the Windows environment.
	- Introduced a nil check for Drag-and-Drop functionality on Windows to improve error handling and robustness.

### CODE REVIEW

File: website/docs/howdoesitwork.mdx [hunk 1]
```diff
@@ -414,20 +414,20 @@ export namespace main {
       this.address = this.convertValues(source["address"], Address);
     }
 
-    convertValues(a: any, classs: any, asMap: boolean = false): any {
+    convertValues(a: any, class: any, asMap: boolean = false): any {
```
Comment (leaanthony) [inline website/docs/howdoesitwork.mdx:417]: I think this one was deliberate as it's a keyword

Comment (deining) [reply website/docs/howdoesitwork.mdx]: > I think this one was deliberate as it's a keyword

I see, reverted this change.

**Comment by leaanthony** [comment]
Thanks 🙏 Please could you add an entry to the changelog located at `website/src/pages/changelog.mdx`?

**Comment by deining** [comment]
> Thanks 🙏 

You are welcome!

> Please could you add an entry to the changelog located at `website/src/pages/changelog.mdx`?

Done!

**Review by leaanthony — APPROVED** [review]

**Pull request merged**